docs/source/tutorial/authentication.rst

```diff
@@ -0,0 +1,93 @@
+.. include:: ../links.inc
+.. _authenticate:
+
+Step 1: Authentication
+======================
+
+``datalad-osf`` needs to communicate with the OSF to create and modify projects under an associated  user account.
+To enable this, the associated user needs to be authenticated using the :command:`osf-credentials` command.
+Therefore, as the very first step, ``datalad osf-credentials`` needs to be ran to authenticate a user.
+Unless credentials expire or change, this command needs to be ran only once per user and system.
+
+.. admonition:: HELP! I'm new to this!
+
+   If this is your reaction to reading the words DataLad dataset, sibling, or dataset publishing,  please head over to the `DataLad Handbook`_ for an introduction to DataLad.
+
+   .. image:: ../_static/clueless.gif
+
+
+Setting credentials
+^^^^^^^^^^^^^^^^^^^
+
+To set credentials, run ``datalad osf-credentials`` anywhere on your system.
+This command prompts for user credentials and stores them internally for OSF operations.
```

Comment:
internally -> in your system's secure credential store

Reply:
done